extensions/walletProviders/kadena/README.md

```diff
@@ -0,0 +1,88 @@
+# 🌍 Kadena Wallet Provider
+
+Kadena Wallet Provider integrates seamlessly with Tatum SDK to provide extended wallet capabilities for Kadena.
+
+## 📖 Description
+
+The Kadena Wallet Provider provides an array of tools for:
+
+- Generating mnemonics for seed phrases.
+- Generating and deriving private keys and addresses from mnemonics.
+- Signing and broadcasting transactions to the Kadena network.
+
+It is built upon popular packages like `@kadena/client`, ensuring a robust and secure foundation.
+
+## 🚀 Quick Start
+
+1. **Installation**
+
+   Firstly, ensure that the `@tatumio/kadena-wallet-provider` package is set as a dependency within your project. Next, import the Kadena Wallet Provider extension:
+
+   ```typescript
+   import { KadenaWalletProvider } from '@tatumio/kadena-wallet-provider'
+   ```
+
+2. **Initialization**
+
+   Create an instance of Tatum SDK passing `KadenaWalletProvider` as one of wallet providers.
+
+   ```typescript
+   const tatumSdk = await TatumSDK.init<Kadena>({
+     network: Network.KADENA,
+     configureWalletProviders: [KadenaWalletProvider],
+   })
+   ```
+
+## 🛠️ How to Use
```

Comment:
Unless I am missing something, the `README` is outdated, no? I did not see any `generateMnemonic()` function or anything of the sort in your `extension` implementation.

Reply:
100%, fixed now

Reply:
Please mention that `getWallet` on its own is not enough as discussed.